src/CalculatorWADTests/CalculatorSession.cs

```diff
@@ -0,0 +1,60 @@
+//******************************************************************************
+//
+// Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Appium.Windows;
+using OpenQA.Selenium.Remote;
+using System;
+using System.Threading;
+
+namespace CalculatorTest
+{
+    public class CalculatorSession
+    {
+        // Note: append /wd/hub to the URL if you're directing the test at Appium
+        private const string WindowsApplicationDriverUrl = "http://127.0.0.1:4723";
+        private const string CalculatorAppId = "Microsoft.WindowsCalculator.Dev_8wekyb3d8bbwe!App";
+        protected static WindowsDriver<WindowsElement> session;
+
+        public static void Setup(TestContext context)
+        {
+            // Launch Calculator application if it is not yet launched
+            if (session == null)
```

Comment:
The guidelines recommended by [Selenium](https://www.seleniumhq.org/docs/03_webdriver.jsp) and Appium require the session creation class to be a static interface instead of an instance, and because WinAppDriver follows the same protocol, it would not be possible to apply "this" or "base" as there is no specific instance of the class to point to.

Reply:
Oops, my bad--I misread this and didn't see "static".